tests/test_github.py

```diff
@@ -960,6 +1163,335 @@ def test_fetch_more_pulls(self):
+                               })
+
+        github = GitHub("zhquan_example", "repo", ["aaa"], exclude_user_data=True)
+
+        with self.assertLogs(logger) as cm:
+            pulls = [pulls for pulls in github.fetch(category=CATEGORY_PULL_REQUEST, from_date=None)]
+
+            self.assertEqual(len(pulls), 2)
+
+            pull = pulls[0]
+            self.assertEqual(pull['origin'], 'https://github.com/zhquan_example/repo')
+            self.assertEqual(pull['uuid'], '58c073fd2a388c44043b9cc197c73c5c540270ac')
+            self.assertEqual(pull['updated_on'], 1451929343.0)
+            self.assertEqual(pull['category'], CATEGORY_PULL_REQUEST)
+            self.assertEqual(pull['tag'], 'https://github.com/zhquan_example/repo')
+            self.assertIsNone(pull['data']['merged_by_data'])
+            self.assertEqual(len(pull['data']['requested_reviewers_data']), 1)
+            self.assertIsNone(pull['data']['requested_reviewers_data'][0])
+            self.assertEqual(len(pull['data']['review_comments_data']), 2)
+            self.assertEqual(len(pull['data']['review_comments_data'][0]['reactions_data']), 0)
+            self.assertEqual(len(pull['data']['review_comments_data'][1]['reactions_data']), 5)
+            self.assertEqual(pull['data']['review_comments_data'][1]['reactions_data'][0]['content'], 'heart')
+            self.assertEqual(len(pull['data']['reviews_data']), 2)
+            self.assertIsNone(pull['data']['reviews_data'][0]['user_data'])
+            self.assertEqual(len(pull['data']['commits_data']), 1)
+
+            pull = pulls[1]
+            self.assertEqual(pull['origin'], 'https://github.com/zhquan_example/repo')
+            self.assertEqual(pull['uuid'], '58c073fd2a388c44043b9cc197c73c5c540270ac')
+            self.assertEqual(pull['updated_on'], 1457113343.0)
+            self.assertEqual(pull['category'], CATEGORY_PULL_REQUEST)
+            self.assertEqual(pull['tag'], 'https://github.com/zhquan_example/repo')
+            self.assertIsNone(pull['data']['merged_by_data'])
+            self.assertEqual(pull['data']['requested_reviewers_data'], [])
+            self.assertEqual(len(pull['data']['review_comments_data']), 4)
+            self.assertEqual(len(pull['data']['review_comments_data'][0]['reactions_data']), 0)
+            self.assertEqual(len(pull['data']['review_comments_data'][1]['reactions_data']), 0)
+            self.assertEqual(len(pull['data']['commits_data']), 1)
+            self.assertEqual(len(pull['data']['reviews_data']), 1)
+            self.assertIsNone(pull['data']['reviews_data'][0]['user_data'])
+            self.assertIsNone(pull['data']['review_comments_data'][0]['user_data'])
+            self.assertIsNone(pull['data']['review_comments_data'][2]['user_data'])
+
+            self.assertEqual(cm.output[0],
+                             "INFO:perceval.backends.core.github:Excluding user data. Won't get user data for the login")
```

Comment:
a minor typo on the log message, that has changed from the initial version of the PR